src/irmin-test/irmin_bench.ml

```diff
@@ -190,7 +191,9 @@ module Make (Store : Irmin.KV with type contents = string) = struct
         >>= fun tree ->
         Store.set_tree_exn v ~info [] tree >|= fun () ->
         if t.clear then Store.Tree.clear tree)
-    >|= fun _ -> Fmt.epr "\n[run done]\n%!"
+    >>= fun _ ->
+    Fmt.epr "\n[run done]\n%!";
+    Store.Repo.close r
```

Comment:
I think it's best to `close` first, just in case it takes a long time.

Reply:
Done, thanks.